release-notes/major10/10.1.0/valtimo-frontend-libraries.md

```diff
@@ -15,6 +15,10 @@ The following features were added:
 
   The option to translate subprocess title is now added. A new key with the value of the `processDefinitionKey` can be used in the translation file to override the default.
 
+* **Moved Audit time**
```

Comment:
This change will go to the 10.2.0 release. 
Please make a PR that goes to branch: `feature/release-notes-10.2.0`